drgn/helpers/linux/net.py

```diff
@@ -45,3 +43,25 @@ def sk_nulls_for_each(head: Object) -> Iterator[Object]:
         "struct sock", head, "__sk_common.skc_nulls_node"
     ):
         yield sk
+
+
+_NETDEV_HASHBITS = 8
+_NETDEV_HASHENTRIES = 1 << _NETDEV_HASHBITS
+
+
+def find_netdev_by_index(net: Object, ifindex: int) -> Object:
```

Comment:
`ifindex` should be `drgn.IntegerLike` instead of `int`, because it's also valid to pass an integer-type `drgn.Object`.